composer.json

```diff
@@ -41,6 +41,7 @@
         "symfony/deprecation-contracts": "^2.1 || ^3",
         "symfony/doctrine-bridge": "^5.4.19 || ^6.0.7 || ^7.0",
         "symfony/framework-bundle": "^5.4 || ^6.0 || ^7.0",
+        "symfony/polyfill-php80": "^1.15",
```

Comment:
I don't think it's worth adding this just for `get_debug_type` which is used in exception message only. Usually var_export is doing good enough job.

Reply:
Sure, but then again, the ORM depends on this package already, so why not use it?

Reply:
fair enough